src/libclipper/src/containers.cpp

```diff
@@ -30,78 +34,191 @@ ModelContainer::ModelContainer(VersionedModelId model, int container_id,
       latency_hist_("container:" + model.serialize() + ":" +
                         std::to_string(replica_id) + ":prediction_latency",
                     "microseconds", HISTOGRAM_SAMPLE_SIZE),
-      avg_throughput_per_milli_(0),
-      throughput_buffer_(THROUGHPUT_BUFFER_CAPACITY) {
+      max_batch_size_(1),
+      max_latency_(0),
+      explore_dist_mu_(.1),
+      explore_dist_std_(.05),
+      budget_decay_(.92),
```

Comment:
Add comments about how these numbers were chosen (even if that is just that you did some offline analysis and picked numbers that worked reasonably well).